truss/remote/baseten/core.py

```diff
@@ -84,14 +84,14 @@ def create_chain(
     chain_name: str,
     chainlets: List[b10_types.ChainletData],
     is_draft: bool = False,
-) -> str:
+) -> Tuple[str, str]:
```

Comment:
Nit/Optional: Ideally we use type aliases to enrich these annotations semantically. Consistent use of type aliases can prevent from accidentally mixing up `chain_id` and `chain_deployment_id` downstream - so even both are low-level strings, the type checker would not treat them as interchangeable. Similar reasoning to why in C++ there's `size_t`...

Reply:
Another option would be something like:

```
class ChainDeploymentHandle(typing.NamedTuple):
  chain_id: str
  chain_deployment_id: str
```

Reply:
Both of these increase readability and coherence a lot and provide much more confidence to any author writing code against this.

Reply:
I should have written

```
from typing import NamedTuple

class ChainDeploymentHandle(NamedTuple):
  ...

```

my bad...